lib/build/dispatcher/dispatcher.js

```diff
@@ -345,7 +345,7 @@ class Dispatcher {
             ? `'${removePathPrefix}'`
             : `''`;
 
-        config.contentToInject = `globalThis.FS_PATH_PREFIX_TO_REMOVE = ${prefix};\n${content}`;
+        config.contentToInject = `globalThis.vulcan.FS_PATH_PREFIX_TO_REMOVE = ${prefix};\n${content}`;
```

Comment:
we must add `globalThis.vulcan = {};` before set `FS_PATH_PREFIX_TO_REMOVE` variable.